apps/web/pages/settings/platform/oauth-clients/create.tsx

```diff
@@ -0,0 +1,75 @@
+import { useRouter } from "next/navigation";
+
+import Shell from "@calcom/features/shell/Shell";
+import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
+import { PERMISSIONS_GROUPED_MAP } from "@calcom/platform-constants/permissions";
+import { showToast } from "@calcom/ui";
+
+import { useCreateOAuthClient } from "@lib/hooks/settings/platform/oauth-clients/usePersistOAuthClient";
+
+import PageWrapper from "@components/PageWrapper";
+import type { FormValues } from "@components/settings/platform/oauth-clients/oauth-client-form";
+import { OAuthClientForm as CreateOAuthClientForm } from "@components/settings/platform/oauth-clients/oauth-client-form";
```

Comment:
I think it's fine to call it `OAuthClientForm` because we use it also to edit oauth clients not only create.